packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts

```diff
@@ -39,37 +33,88 @@ export default class SchedulePaymentFormValidator {
   get isValid() {
     return (
       this.isPaymentTypeValid &&
-      this.isRecipientValid &&
+      this.isRecipientAddressValid &&
       this.isAmountValid &&
       this.isGasTokenValid &&
       this.isMaxGasFeeValid
```

Comment:
Thoughts re validation of dates? Currently I can choose a one-time payment in the past or an until in the past for recurring.

Reply:
Yeah, we should add validation for this, or a minimum DateTime to the picker.

Reply:
captured this in Linear